doc/classes/NodePath.xml

```diff
@@ -65,8 +65,22 @@
 				[/codeblock]
 			</description>
 		</constructor>
+		<constructor name="NodePath">
+			<return type="NodePath" />
+			<param index="0" name="from" type="StringName" />
+			<description>
+				Constructs a NodePath from a [StringName]. This method will be faster than constructing from a String if the StringName does not contain any slashes or colons.
```

Comment:
```suggestion
				Constructs a NodePath from a [StringName]. This method will be faster than [code skip-lint]NodePath(from)[/code] if the StringName does not contain any slashes or colons.
```
It mentions constructing, and then says faster than constructini, I think this is clearer, or something like
```suggestion
				Constructs a NodePath from a [StringName]. This method will be faster than constructing directly from a String if the StringName does not contain any slashes or colons.
```
Same below

Reply:
It's saying that constructing *from a StringName* is faster than constructing *from a String*. Being direct or not isn't a meaningful distinction, as doing things indirectly is implied to be slower than directly. Writing `NodePath(from)` is inaccurate because that will be equally fast as of @dalexeev's suggestion.

Reply:
Gotcha yes that makes sense was thinking it was another side of the conversion 🙂